include/vcpkg/base/contractual-constants.h

```diff
@@ -207,6 +207,7 @@ namespace vcpkg
     inline constexpr StringLiteral SwitchDebug = "debug";
     inline constexpr StringLiteral SwitchDebugBin = "debug-bin";
     inline constexpr StringLiteral SwitchDebugEnv = "debug-env";
+    inline constexpr StringLiteral SwitchDeleteFileAfterUpload = "delete-file-after-upload";
```

Comment:
Since this is only used for z-upload-metrics can we just make that command just do that rather than adding a switch?